R/get_stan_data.R

```diff
@@ -173,6 +174,20 @@ get_stan_data <- function(input_count_data,
 
   inf_to_count_delay_max <- length(inf_to_count_delay)
 
+
+  # If dist_matrix null use independence correlation and update flag
+  if (is.null(dist_matrix)) {
+    ind_corr_func <- 1
+    # This dist_matrix will not be used, only needed for stan data specs.
+    dist_matrix <- matrix(
+      0,
+      nrow = subpop_data$n_subpops - 1,
```

Comment:
The trick you are using here is that you are just passing in dummy data to stan that won't be used (but has the correct dimensions) right? Think this is a good approach!